### ISSUE TITLE
CSS/Contrast Accessibility Violations

### ISSUE THREAD

**Issue by cwyckoff3** [IBM/kui#3075]
Accessibility Violations 

Error response text has a contrast of 4.38, requirement is 4.5.

![Screen Shot 2019-10-22 at 1 58 03 PM](https://user-images.githubusercontent.com/9663734/67315209-1d1a0780-f4d4-11e9-921b-f577af68d3d3.png)

YAML tab has low contrast when hovering over links
kubectl get applications --all-namespaces
[Link to git issue](https://github.ibm.com/IBMPrivateCloud/roadmap/issues/31730)
Can text color be changed to #171717?
